server/schedulers/balance_test.go

```diff
@@ -170,9 +170,9 @@ func (s *testBalanceLeaderSchedulerSuite) TestBalanceLimit(c *C) {
 	c.Check(s.schedule(nil), IsNil)
 
 	// Stores:     1    2    3    4
-	// Leaders:    10   0    0    0
+	// Leaders:    16   0    0    0
 	// Region1:    L    F    F    F
-	s.tc.updateLeaderCount(1, 10)
+	s.tc.updateLeaderCount(1, 16)
```

Comment:
Why change to 16?

Reply:
note that: shouldBalance() uses math.Min(sourceSizeDiff, targetSizeDiff) >= float64(region.ApproximateSize)*tolerantRatio to decide whether to balance.

Before when size is zero, score is zero. for this case, 
// Stores:     1    2    3    4
// Score:    100   0    0    0		
avgScore is 25, and sizediff satisfy the formula above just enough

After when size is zero, score is 1, so for this case
// Stores:     1    2    3    4
// Score:    100   1    1    1	
avgScore is 25.75, and sizediff doesn't satisfy the formula above 

so enlarge the leader count

Reply:
got it